src/portable/microchip/pic32mz/dcd_pic32mz.c

```diff
@@ -163,6 +163,10 @@ void dcd_remote_wakeup(uint8_t rhport)
   USB_REGS->POWERbits.RESUME = 1;
 #if CFG_TUSB_OS != OPT_OS_NONE
   osal_task_delay(10);
```

Comment:
The usb specs state the delay must be within 1-15 ms. Other mcu is doing well with 1 ms, maybe we should reduce it to 1-2 ms only, 10ms is a bit overkill, would it pass the cv test with 1ms of delay ? 

![image](https://user-images.githubusercontent.com/249515/149953052-f2b99ecb-0323-4521-86ba-c4844c3cc052.png)

Reply:
just do a quick check with musb spec, it does indeed say 10ms as optimum value (2-15 ms). So I think 10m here make sense 
![image](https://user-images.githubusercontent.com/249515/150054647-bcc8b9f3-34a0-424f-8423-7a59bf3f11ef.png)